docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -2440,215 +2440,106 @@ And can be defined using different criteria like:
 
-.EXAMPLE: create a security violaton if the SSID is OPEN and the owner is igmout
-[source,ini,numbered]
-----
-  [igmout]
-  filter = owner.pid
-  operator = is
-  value = igmout
+.EXAMPLE_Reject_specific_MAC will filter a MAC address and reject it by assigning the role REJECT.
 
-  [open]
-  filter = ssid
-  operator = is
-  value = OPEN
+.EXAMPLE_Detect_VOIP will auto set Avaya and Polycom as phones by matching vendor MAC, and set to default role
 
-  [2:igmout&ssid]
-  scope = RegisteredRole
-  action = trigger_security_event
-  action_param = mac = $mac, tid = SOMETHING_HAPPENED, type = CUSTOM
-----
+.EXAMPLE_Reject_User_Unless_Machine will refuse user authentication without prior machine authentication
 
-.EXAMPLE: autoregister the device and assign the role staff to each device where the username is igmout.
-[source,ini,numbered]
-----
-  [igmout]
-  filter = username
-  operator = is
-  value = igmout
+.EXAMPLE_Autoregister_Printer_Scanner will autoregister printers and scanners and add a note.
 
-  [secure]
-  filter = ssid
-  operator = is
-  value = SECURE
+You can have a look in the file `vlan_filters.conf`, there are some examples on how to use and define filters.
 
-  [3:igmout&secure]
-  scope = AutoRegister
-  role = staff
+Advanced LDAP Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  [4:igmout&secure]
-  scope = NodeInfoForAutoReg
-  role = staff
-----
+ldapfilter action
+^^^^^^^^^^^^^^^^^
 
-.EXAMPLE: autoregister all Windows devices and assign them the role default.
-[source,ini,numbered]
-----
-  [Windows_AutoReg]
-  filter = fingerbank_info.device_hierarchy_names
-  operator = includes
-  value = Windows
+ldapfilter action override the internal ldap filter packetfence create internally (uid=$username) so you can create a custom filter that match your needs.
 
-  [AUTOREG_WINDOWS:Windows_AutoReg]
-  scope = AutoRegister
-  role = default
+Per example something like this (search for the user and check to see if it's permitted based on some criteria):
 
-  [WINDOWS_ROLE:Windows_devices]
-  scope = NodeInfoForAutoReg
-  role = default
-----
+  (&(|(cn=${radius_request.Stripped-User-Name})(cn=${radius_request.User-Name}))(|(permitWifi=*)(grade=staff)(memberOf=CN=WifiGroup,OU=Security Groups,DC=ad,DC=acme,DC=com)))
 
-.EXAMPLE: filter a MAC address and reject it by assigning the role REJECT.
-[source,ini,numbered]
-----
-  [MAC]
-  filter = node_info.mac
-  operator = match
-  value = aa:bb:cc:dd:ee:ff
+Set_role_on_not_found action
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-  [SSID_Open]
-  filter = ssid
-  operator = is
-  value = Open
+set_role_on_not_found is a way to define a role is the rule doesn't match, let's take the ldapfilter example above.
 
-  [REJECT:MAC&SSID_Open]
-  scope = RegistrationRole
-  role = REJECT
-----
+If we add the action set_role_on_not_found = REJECT so it mean that the device will be rejected if the ldap filter doesn't return anything. (if it match then set_role action will be applied)
```

Comment:
so it mean -> so it means
ldap filter -> LDAP filter
if it match -> if it matches